.github/workflows/ur-source-checks.yml

```diff
@@ -53,7 +53,6 @@ jobs:
         -DCMAKE_BUILD_TYPE=Debug
         -DUR_BUILD_TESTS=OFF
         -DUR_FORMAT_CPP_STYLE=ON
-        -DUMF_DISABLE_HWLOC=ON
```

Comment:
Is it not faster to just fetch hwloc via apt-get rather than building it ourself?

Reply:
That's a good point, I'll add the package.